status-im/status-react
# [#6815] restyle sent-transaction screen

fixes #6815 

### Summary:

This restyles the Wallet's final transaction sent screen. This was done according the `Transaction Sent` screen in the following figma.

https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction?node-id=0%3A1

### Review notes:

I have taken much care in this commit to make the code clear and meaningful.  The reviewer will see that I have inlined styles in quite a few instances. It is my opinion that abstractions like `component/flex` which means `{:flex 1}` are detrimental to the understanding of the code. When abstracting and naming things they need to merit abstraction and provide value  (i.e `map` is a good abstraction and `components/flex` isn't). If an abstraction is just a name that doesn't mean anything, all that name does is remove meaning from the immediate and it obfuscates it.

Inlining simple styles so that it helps the reader understand why this `react/view` component is being used is very very helpful.

I have also inlined style that creates a graphical figure because that is the full meaning of the code to abstract that removes meaning from the location where you are actually looking for it.

If a style removes meaning from the view or if it is complex and reused I say yes abstract it, and if you do abstract it give it a meaningful name.

### Testing notes:

Just ensure that the Transaction sent screen is displayed as it is in the figma file.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted (optional)
**Functional**
- wallet / transactions

### Steps to test:
- Open Status
- Send a transaction and view the final fabulous Transaction Sent screen
- Hit the done button and see that it behaves correctly

status: ready


<blockquote><img src="/file/SIeZPHvrX7KUwEBW8VQct8C5/thumbnail?ver=thumbnails/07cb8fe4-f735-4817-8a2f-c90f8d72fa38" width="48" align="right"><div><img src="https://static.figma.com/app/icon/1/favicon.svg" height="14"> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>
